lib/msal-browser/src/config/Configuration.ts

```diff
@@ -74,6 +75,10 @@ export type BrowserAuthOptions = {
      * Enum that represents the protocol that msal follows. Used for configuring proper endpoints.
      */
     protocolMode?: ProtocolMode;
+    /**
+     * Enum that configures options for the OIDC protocol mode.
+     */
+    OIDCOptions?: OIDCOptions | null;
```

Comment:
This probably doesn't need `null` as an option since `undefined` is already acceptable as well